AppHandling/Start-NavContainerAppDataUpgrade.ps1

```diff
@@ -18,39 +18,64 @@ function  Start-BcContainerAppDataUpgrade {
     Param (
         [string] $containerName = $bcContainerHelperConfig.defaultContainerName,
         [string] $tenant = "default",
-        [Parameter(Mandatory=$true)]
+        [Parameter(Mandatory = $true)]
         [string] $appName,
         [string] $appVersion = "",
-        [string] $language = ""
+        [string] $language = "",
+        [string] $exclusiveAccessTicket = "",
+        [string] $path,
+        [switch] $force,
+        [switch] $skipVersionCheck,
+        [ValidateSet("Add", "Clean", "Development", "ForceSync", "None")]
+        [string] $syncMode
     )
 
-$telemetryScope = InitTelemetryScope -name $MyInvocation.InvocationName -parameterValues $PSBoundParameters -includeParameters @()
-try {
-
-    Invoke-ScriptInBcContainer -containerName $containerName -ScriptBlock { Param($appName, $appVersion, $tenant, $language)
-        Write-Host "Upgrading app $appName"
-        $parameters = @{
-            "ServerInstance" = $ServerInstance;
-            "Name" = $appName;
-            "Tenant" = $tenant
-        }
-        if ($appVersion) {
-            $parameters += @{ "Version" = $appVersion }
+    $telemetryScope = InitTelemetryScope -name $MyInvocation.InvocationName -parameterValues $PSBoundParameters -includeParameters @()
+    try {
+        $containerPath = $null

```

Comment:
use empty string instead of null - inside the container it will be empty (not null)

Reply:
OK